bindings/rust-bindings/src/lib.rs

```diff
@@ -54,7 +54,7 @@ mod tests {
     fn test_read_chain_configs() {
         let mainnet = SUPERCHAINS.get("mainnet").unwrap();
         assert_eq!(mainnet.config.name, "Mainnet");
-        assert_eq!(mainnet.chain_ids.len(), 8);
+        assert_eq!(mainnet.chain_ids.len(), 9);
```

Comment:
@refcell I think this is going to trip us up every time a new mainnet chain is added. Can we remove this assertion or replace it with something different?